simpleclient_log4j/src/main/java/io/prometheus/client/log4j/InstrumentedAppender.java

```diff
@@ -0,0 +1,93 @@
+package io.prometheus.client.log4j;
+
+import io.prometheus.client.CollectorRegistry;
+import io.prometheus.client.Counter;
+import org.apache.log4j.Appender;
+import org.apache.log4j.AppenderSkeleton;
+import org.apache.log4j.Level;
+import org.apache.log4j.spi.LoggingEvent;
+
+
+public class InstrumentedAppender extends AppenderSkeleton {
+    
+	private final CollectorRegistry registry;
+
+    private Counter all;
+    private Counter trace;
+    private Counter debug;
+    private Counter info;
+    private Counter warn;
+    private Counter error;
+    private Counter fatal;
+
+    /**
+     * Create a new instrumented appender using the default registry.
```

Comment:
Yeah, you're right. I'm pretty sure its a singleton so only single instance but best to avoid headaches I suppose. It makes testing easier when its configurable